examples/ethereum/rosetta-cli-conf/mainnet/exempt_accounts.json

```diff
@@ -0,0 +1,20 @@
+[
+   {
+        "account_identifier": {
+          "address":"0x0000000000000000000000000000000000000000"
```

Comment:
possible to add comment somewhere why we exclude this

Reply:
will do, probably it's better to add a private network, instead of mainnet